File: RestApi/src/main/java/com/apon/service/valueobject/StudentVOGet.java
package com.apon.service.valueobject;

import com.apon.service.valueobject.database.StudentDVO;
import com.apon.service.valueobject.database.VolunteerMatchDVO;

import java.util.List;

public class StudentVOGet {
    private StudentDVO studentDVO;
    private List<VolunteerMatchDVO> volunteerMatchDVOS;

    public StudentDVO getStudentDVO() {
        return studentDVO;
    }

    public void setStudentDVO(StudentDVO studentDVO) {
        this.studentDVO = studentDVO;
    }

    public List<VolunteerMatchDVO> getVolunteerMatchDVOS() {
        return volunteerMatchDVOS;
    }

    public void setVolunteerMatchDVOS(List<VolunteerMatchDVO> volunteerMatchDVOS) {
        this.volunteerMatchDVOS = volunteerMatchDVOS;
    }
}

File: Angular/src/app/valueobject/student.vo.get.ts
import {StudentDVO} from "./dvo/student.dvo";
import {VolunteerMatchDVO} from "./dvo/volunteermatch.dvo";

export class StudentVOGet {
  public studentDVO: StudentDVO = new StudentDVO();
  public volunteerMatchDVOS: VolunteerMatchDVO[] = [];
}
